### PR TITLE
Feature/mnv flags

### PR DESCRIPTION
**Pull request by davidrajones**

### CODE REVIEW

**Review by rulixxx — CHANGES_REQUESTED** [review]

**Comment by rulixxx** [comment]
Nothing major :

Some indentation issues.
I noticed you updated some of the licences ( not sure how that is handled). Some files seem to have the old date.

Don't forget to update the README with the new option

Otherwise code is pretty good, your use of references in perl is quite impressive! ( Had to lookup some things ;)  )

Have a good easter holiday

File: bin/cgpFlagCaVEMan.pl [hunk 1]
```diff
@@ -646,7 +660,7 @@ sub runFlagger{
     }
     return -1;
     }elsif($flagName eq 'cavemanMatchNormalProportionFlag'){
-        return $flagger->getCavemanMatchedNormalResult($vcf->get_column($x,$VCF_COLUMN_NORMAL),$vcf->get_column($x,$VCF_COLUMN_TUMOUR),$vcf->get_column($x,$VCF_COLUMN_FORMAT));
+        return $flagger->getCavemanMatchedNormalResult($vcf,$x,$index);
```
[the remark this answers is not in the document]
Comment (davidrajones) [reply bin/cgpFlagCaVEMan.pl:663]: d588932

File: t/3_cgpFlagCaVEMan.t [hunk 2]
```diff
@@ -281,4 +323,5 @@ sub remove_created_files{
 	#config created
 	unlink($test_config);
 	unlink($test_out_oldVersionVCF);
+  unlink($test_out_mnv_file);
```
[the remark this answers is not in the document]
Comment (davidrajones) [reply t/3_cgpFlagCaVEMan.t:326]: a113854

File: t/5_genomePostProcessor.t [hunk 3]
```diff
@@ -53,255 +53,255 @@ subtest 'Initialise module (no params)' => sub {
...
 subtest 'Initialise module (ALL params)' => sub {
-	my $processor = new_ok('Sanger::CGP::CavemanPostProcessor::GenomePostProcessor' => [
-																			'tumBam' => $T_BAM,
-																			'normBam' => $T_BAM,
-																			'minDepthQual' => 2,
+  my $processor = new_ok('Sanger::CGP::CavemanPostProcessor::GenomePostProcessor' => [
+                                      'tumBam' => $T_BAM,
+                                      'normBam' => $T_BAM,
+                                      'minDepthQual' => 2,
                                                                             'depthCutoffProportion' => (1/2),
```
[the remark this answers is not in the document]
Comment (davidrajones) [reply t/5_genomePostProcessor.t]: 97a2d8f

File: t/4_exomePostProcessor.t [hunk 4]
```diff
@@ -55,123 +55,124 @@ subtest 'Initialise module (no params)' => sub {
...
-																			'maxPhasingMinorityStrandReadProportion' => 0.01,
-																			'maxMatchedNormalAlleleProportion' => 0.08,
-																			'readPosBeginningOfReadIgnoreProportion' => 0.03,
-																			'readPosTwoThirdsOfReadExtendProportion' => 0.07,
-																			'minRdPosDepth' => 10]);
+  my $processor = new_ok('Sanger::CGP::CavemanPostProcessor::ExomePostProcessor' => [
+                          'tumBam' => $T_BAM,
+                          'normBam' => $T_BAM,
+                          'minDepthQual' => 2,
+                          'depthCutoffProportion' => (1/2),
```
[the remark this answers is not in the document]
Comment (davidrajones) [reply t/4_exomePostProcessor.t:123]: 97a2d8f

**Pull request merged**